index.html

```diff
@@ -10773,43 +10773,70 @@ <h3>Determining common capabilities</h3>
       This section provides an example of what such a library
       function might do.</p>
       <pre class="example highlight">
-RTCRtpParameters function myCapsIntersection(RTCRtpCapabilities sendCaps,
-   RTCRtpCapabilities remoteRecvCaps) {
-// Function returning the intersecting RTCRtpParameters, based on
-// the sender and remote receiver capabilities. The goal is to enable
-// a call with one audio and one video track with minimum fuss.
+RTCRtpCapabilities function getCommonCapabilities(RTCRtpCapabilities localCapabilities,
+   RTCRtpCapabilities remoteCapabilities) {
+// Function returning the common capabilities, based on the local sender and
+// remote receiver capabilities. An implementation is available here:
+// https://webrtc.github.io/adapter/adapter-latest.js
 //
-// Steps to be followed to determine the features that the receiver
-// and sender have in common:
-// 1. Determine the codecs that the sender and receiver have in common.
-// 2. For each common codec, determine the common formats, header
-//    extensions and RTCP feedback mechanisms.
-// 3. Determine the common robustness (forward error correction and 
-//    retransmission) mechanisms. 
-// 4. Determine the payloadType to be used, based on the receiver
+// Steps to determine the common capabilities of the local sender and
+// remote receiver:
+// 1. Determine the common codecs.
+// 2. For each common codec, determine the common header extensions and
+//    RTCP feedback mechanisms.
+// 3. For each common audio codec, determine the minimum number of
+//    channels supported.
+// 3. For each common codec, determine the common parameters, such as:
+//    a. For H.264/AVC, the minimum profile-level-id and common
+//       packetization-mode.
+//    b. For Opus, usedtx and useinbandfec set to one if both sides support
+//       it, otherwise zero.
+// 4. Determine the common robustness (forward error correction and 
+//    retransmission) mechanisms.
+// 5. Determine the payloadType to be used, based on the receiver
```

Comment:
I think its the offerers payloadType, not the receivers

Reply:
We have the local sender's preferred payload type and the remote receiver's preferred payload type.  Shouldn't the sender send with the receiver's preferred type? 

Remember, in an Offer, the PT represents the payload type the Offerer would like to receive. The Answer contains the PT the remote receiver wants to receive (and which the local sender should send).